sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -277,22 +331,69 @@ public Binding connectToService(FlutterView view, Core core, MessagePipeHandle p
         });
     }
 
+    /**
+     * Initialize our Flutter config values by searching for a .properties file
+     * containing properties for our configuration variables. If no such
+     * properties file is found in our assets folder, the configuration
+     * variables will retain their default values.
```

Comment:
Let's discuss so I can understand the use case.  I'm skeptical that reading more data from flash (particularly non-sequential data) and parsing a config file has zero overhead at startup.  We would ideally like to improve our startup by 10x, which mean this sort of thing looks small today but will look large in the future.